workloads/templates/common-templates.md

```diff
@@ -31,14 +31,16 @@ $ oc create -f https://github.com/kubevirt/common-templates/releases/download/$V
 ## Editable fields
 
 You can edit the fields of the templates which define the amount of resources which the VMs will receive.
-The list of fields includes:
 
-- spec.template.spec.domain.cpu.cores
-- spec.template.spec.domain.resources.requests.memory
-- spec.template.spec.domain.devices.disks
-- spec.template.spec.volumes
-- spec.template.spec.networks
+Each template can list a different set of fields that are to be considered editable.
+The fields are used as hints for the user interface, and also for other components in the cluster.
 
+The editable fields are taken from annotations in the template:
+```yaml
+metadata:
+  annotations:
+    template.cnv.io/editable: ...
+```
```

Comment:
Please leave an example path there (cpu and memory perhaps) and explain what the one per line paths mean (described here: https://github.com/kubevirt/common-templates/blob/master/templates/README.md#templateyaml). Jsonpath should be a link and point to https://kubernetes.io/docs/reference/kubectl/jsonpath/ I think.

> The content is a line separated list of jsonpath selectors.
> The jsonpath root is the objects: element of the template

Reply:
done